File: Intangic.RealPropertyTaxProjectionServices/Intangic/Utils/ConfigurationHelper.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

using System.Configuration;

namespace Intangic.Utils
{
    public class ConfigurationHelper
    {
        public static string GetString(string key) {
            string value = string.Empty;
            try {
                value = ConfigurationManager.AppSettings[key];
            }
            catch (Exception) {
                //do nothing
            }
            return value;
        }
    }
}

File: RealPropertyTaxProjectionVB/Intangic/Utils/ConfigurationHelper.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks
Imports System.Windows.Forms

Imports System.Configuration


Namespace Intangic.Utils

    Public Class ConfigurationHelper

        Public Shared Function GetString(ByVal key As String) As String
            Dim value As String = Nothing
            Try
                'value = ConfigurationManager.AppSettings(key)
                value = My.Settings.Item(key)
            Catch ex As Exception
                'do nothing
            End Try

            Return value
        End Function

    End Class

End Namespace
